website/server/libs/payments.js

```diff
@@ -125,13 +130,39 @@ api.addSubToGroupUser = async function addSubToGroupUser (member, group) {
         {name: 'EMAIL', content: getUserInfo(member, ['email']).email},
         {name: 'PAYMENT_METHOD', content: memberPlan.paymentMethod},
         {name: 'PURCHASED_PLAN', content: JSON.stringify(memberPlan)},
-        {name: 'ACTION_NEEDED', content: 'User has joined group plan. Tell them to cancel subscription then give them free sub.'},
+        {name: 'ACTION_NEEDED', content: 'User has joined group plan and has been told to cancel their subscription then email us. Ensure they do that then give them free sub.'},
+        // TODO User won't get email instructions if they've opted out of all emails. See if we can make this email an exception and if not, report here whether they've opted out.
       ]);
     }
 
-    if ((ignorePaymentPlan || ignoreCustomerId) && !customerHasCancelledGroupPlan) return;
+    if ((ignorePaymentPlan || ignoreCustomerId) && !customerHasCancelledGroupPlan) {
+      // member has been added to group plan but their subscription will not be changed
+      // automatically so they need a special message in the email
+      if (memberPlan.paymentMethod === this.constants.GOOGLE_PAYMENT_METHOD) {
+        previousSubscriptionType = 'Google';
```

Comment:
Hmm. I have the same fear of the strings changing, that is why I want the constants :P

I was thinking this:
```
 if (memberPlan.paymentMethod === this.constants.GOOGLE_PAYMENT_METHOD || memberPlan.paymentMethod === this.constants.IOS_PAYMENT_METHOD) {
        previousSubscriptionType = this.constants.GOOGLE_PAYMENT_METHOD;
      } 
```

And, in theory, if we correctly use the `this.constants.GOOGLE_PAYMENT_METHOD` constant everywhere, then when it changes it one place it will change in all. Unless I am missing something.